tool/ood-gen/lib/tutorial.ml

```diff
@@ -35,6 +35,15 @@ type metadata = {
 }
 [@@deriving of_yaml]
 
+type document = {
+  title : string;
+  category : string;
+  section_heading : string;
+  content : string;
+  slug : string;
+}
+[@@deriving of_yaml, show { with_path = false }]
```

Comment:
I think we don't need of_yaml here because we are not parsing the search documents from YAML